cypress/integration/vending-machine.spec.js

```diff
@@ -0,0 +1,103 @@
+import {
+  EMPTY_PRODUCT_NAME,
+  EMPTY_PRODUCT_NUMBER,
+  EMPTY_PRODUCT_PRICE,
+  INVALID_PRODUCT_NUMBER_MINIMUM,
+  INVALID_PRODUCT_PRICE,
+  INVALID_PRODUCT_PRICE_MINIMUM,
+} from '../../src/js/constants.js';
+
+const BASE_URL = '../../index.html';
+
+const addProduct = (name, price, number, clickEvent) => {
+  cy.get('#product-name-input').type(name);
+  cy.get('#product-price-input').type(price);
+  cy.get('#product-quantity-input').type(number);
+  cy.get('#product-add-button')
+    .click()
+    .then(() => clickEvent);
```

Comment:
```suggestion
  return cy.get('#product-add-button')
    .click()
```
이렇게 하면 callback을 받지 않아도 될 것 같아요!